src/System.ServiceProcess.ServiceController/ref/Configurations.props

```diff
@@ -2,8 +2,9 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netcoreapp;
```

Comment:
No I will remove it and just leave netstandard and netfx given we have to have a netstandard ref now for compat with the older package.